common/lib/xmodule/xmodule/course_module.py

```diff
@@ -1244,7 +1244,7 @@ def get_discussion_blackout_datetimes(self):
                     raise ValueError
             return ret
         except (TypeError, ValueError):
-            log.exception(
+            log.info(
                 "Error parsing discussion_blackouts %s for course %s",
```

Comment:
Is this mostly because course teams put 2099-11-20 in here to ensure the forums stay permanently blocked off?